launcher-package/citest/test.bat

```diff
@@ -8,6 +8,7 @@ unzip ..\target\universal\sbt.zip -d freshly-baked
 SETLOCAL
 
 "freshly-baked\sbt\bin\sbt" about
+"freshly-baked\sbt\bin\sbt" -Dsbt.repository.config=.\repositories -Dsbt.version=0.100 about

```

Comment:
I think `=` sign goes way without quotation:

```suggestion
"freshly-baked\sbt\bin\sbt" "-Dsbt.repository.config=.\repositories" "-Dsbt.version=0.100" about
```

Reply:
Thanks 👍🏽 will try this now